app/components/LaptopMockup.tsx

```diff
@@ -29,7 +29,7 @@ const LaptopMockup = ({ imageSrc, isHomepage = false }: MockupProps) => {
         <Image
           src={imageSrc}
           alt="desktop image of project"
-          objectFit="cover"
+          style={{ objectFit: "fill" }}
```

Comment:
The width and height being passed are serving as placeholders in a way... If you will be in office hours today, we can look at it and try another approach. I did reach out to Mentor Santiago and he did verify that "_Internal static images in the public folder don’t need height & width set , as next sets this on its own because it can read them at runtime (unless your eslint settings prevent you from doing this), but images from external APIs can’t be read at runtime and so next requires a height and width to explicitly be set on those._"

     ```
<Image
        fill
        src="/example.png"
        sizes="(max-width: 768px) 100vw, (max-width: 1200px) 50vw, 33vw"
      />
```